jest.config.integration.js

```diff
@@ -36,4 +36,5 @@ module.exports = {
     ],
   ],
   setupFilesAfterEnv: ['<rootDir>/packages/kbn-test/target/jest/setup/after_env.integration.js'],
+  coverageDirectory: '<rootDir>/target/jest-coverage/jest_integration',
```

Comment:
We need to copy json file from this path into the temp directory next to the other jest coverage json before merging: nyc [does not support](https://github.com/istanbuljs/nyc/issues/879) glob patterns or list of folders, so our [config](https://github.com/elastic/kibana/blob/master/src/dev/code_coverage/nyc_config/nyc.jest.config.js#L23-L24) uses 
a single folder with jest oss, xpack and integration coverage json files

Reply:
Probably it can added in [src/dev/code_coverage/shell_scripts/merge_jest_and_functional.sh](https://github.com/elastic/kibana/blob/6671cf3c2255c89bc1d2913e62bd06a59d2bd2df/src/dev/code_coverage/shell_scripts/merge_jest_and_functional.sh#L5) :
```
cp -r tmp/extracted_coverage/target/kibana-coverage/jest_integration/json-summary.json target/kibana-coverage/jest/integration-json-summary.json  
```

Reply:
I also just changing path to `<rootDir>/target/kibana-coverage/jest_integration` to follow the same pattern `/target/kibana-coverage/**`